.evergreen/config.yml

```diff
@@ -1220,6 +1269,21 @@ buildvariants:
   tasks:
     - ".atlas-connect"
 
+- matrix_name: "load-balancer-all"
+  matrix_spec:
+    os-fully-featured:
+      - "ubuntu-18.04"
```

Comment:
haproxy isn't installed by default on macOS like it is on these hosts, and other drivers seem to have skipped running these tests on macOS, so I think just testing on Ubuntu should be sufficient.